packages/@aws-cdk/aws-codepipeline-actions/test/integ.cfn-template-from-repo.lit.expected.json

```diff
@@ -59,10 +60,7 @@
               "Effect": "Allow",
               "Principal": {
                 "AWS": {
-                  "Fn::GetAtt": [
-                    "PipelineRoleD68726F7",
-                    "Arn"
-                  ]
+                  "Fn::GetAtt": ["PipelineRoleD68726F7", "Arn"]
```

Comment:
What's going on here? Why did the formatting change? These expectation files are supposed to be formatted automatically when running "cdk run integ".

Please revert any changes to the expectation files were not produced by this change. It's important so we can review.

Reply:
My bad there, I thought I turned off when saving these files, should be back now.